handout/components/creating_components.md

```diff
@@ -1,9 +1,11 @@
 # Creating Components
 
-Components in Angular 2 build upon this idea. We define a component's application logic inside a class. To this we then attach a selector and a template.
+Components in Angular 2 build upon the lessons learned from Angular 1.5. We define a component's application logic inside a class. To this we attach metadata using a TypeScript **Decorator**, which is simply a function call that allows the specification of the **selector** and **template**.
```

Comment:
Decorator does not need to be capitalized. A decorator is a more generic concept than just allowing `selector` and `template`. This should say something like "a decorator allows you to modify a class or function definition and add metadata to properties and function arguments." Then say something about `selector` and `template`.